plugins/template.md

```diff
@@ -212,7 +212,7 @@ And that will be changed to:
 
 ### `template_preview_replace_values`
 
-This is an object containing items that will be replaced with their respective string values in the template preview shown in the template dialog.
+This is an object containing items that will be replaced with their respective string values in the template preview shown in the template dialog, **but will not be replaced when a template is inserted into the editor content**.
```

Comment:
please change the comma to a semi-colon, otherwise it's just one very long sentence (the original sentence was already kinda pushing it)